common/styles/components/_opening_hours.scss

```diff
@@ -80,42 +80,43 @@
 }
 
 .opening-hours__table {
+  max-width: 600px;
   color: color('silver');
   width: 100%;
   margin-bottom: 2 * $vertical-space-unit;
 
-  .opening-hours--light & {
+  &.opening-hours--light {
```

Comment:
Shouldn't this be `opening-hours__table--light` if it's going to be a sibling of `opening-hours__table`?